helm_deploy/hmpps-locations-inside-prison-api/values.yaml

```diff
@@ -51,3 +51,6 @@ generic-service:
 
 generic-prometheus-alerts:
   targetApplication: hmpps-locations-inside-prison-api
+
+cron:
+  training_reset_schedule: ""
```

Comment:
This may result in a failed deployment.

Reply:
How do I exclude?

Reply:
better way would be to set this param in the cronjob and control it with a helm value.
```
   suspend: true  # This suspends the CronJob
```